src/examples/pnpm.yml

```diff
@@ -0,0 +1,13 @@
+description: >
+  Runs Cypress tests using pnpm.
+  Installs dependencies and caches NPM modules and the Cypress binary.
+usage:
+  version: 2.1
+  orbs:
+    cypress: cypress-io/cypress@3
+  workflows:
+    use-my-orb:
+      jobs:
+        - cypress/run:
+            package-manager: "pnpm"
+            start-command: "pnpm start"
```

Comment:
Let's add the `install-command` argument to this example as well: `pnpm install --frozen-lockfile`

Reply:
good idea 👍

Reply:
added now